client/landing/gutenboarding/index.tsx

```diff
@@ -172,3 +178,29 @@ function ensureRandomizedDesignsAreUpToDate() {
 function isDeepEqual( stored: Design[], available: Design[] ): boolean {
 	return isEmpty( xorWith( stored, available, isEqual ) );
 }
+
+/**
+ * Pre-fill Anchor.fm podcast title if available
+ */
+async function setPodcastTitle() {
+	// Feature flag 'anchor-fm-dev' is required for anchor podcast id to be read
+	if ( ! config.isEnabled( 'anchor-fm-dev' ) ) {
+		return;
+	}
+
+	const anchorFmPodcastId = new URLSearchParams( window.location.search )
+		.get( 'anchor_podcast' )
+		?.replace( /[^a-zA-Z0-9]/g, '' );
```

Comment:
This is replicated code from path.ts because we can't use hooks here. Maybe we can export a sanitizing function.

Reply:
Might be possible to also solve this duplication when it's moved over to a `usePodcastTitle()` hook https://github.com/Automattic/wp-calypso/pull/48430#issuecomment-754632085.